### PR TITLE
prep-javascript-objects - Christy Pessagno

### PR DESCRIPTION
**Pull request by christy-pessagno**
CleanShot 2024-03-07 at 16 51 04@2x

### CODE REVIEW

File: prep-javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const person = {
+  firstName: 'Christy',
+  lastName: 'Pessagno',
+  hobby: 'Photography',
+  job: 'Filmmaker',
+  previousJob: 'Editor',
```
Comment (BrettAlbright) [inline prep-javascript-objects/main.js]: you want to add these properties after creating the object. 

one should be added using dot notation
one should be added using bracket notation. 

We want to make sure you get practice with modifying existing objects.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Just a couple small updates are noted!

File: prep-javascript-objects/main.js (hunk 1 above)
Comment (christy-pessagno) [reply prep-javascript-objects/main.js]: I'll update after I dig back in to the reading a bit. Thanks!

**Comment by christy-pessagno** [comment]
CleanShot 2024-03-07 at 17 11 58@2x

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**